### DISCUSSION TITLE
I'm curious about some options of Indexing command

### DISCUSSION THREAD

**Discussion by ikiki1008** [oracle/opengrok#4163] [Q&A]
Hi, I just wanna ask about the options used by indexing command 
here is what I'm curious about

java \
    -Djava.util.logging.config.file=/opengrok/etc/logging.properties \
    -jar /opengrok/dist/lib/opengrok.jar \
    -c /usr/local/bin/ctags \
    -s /opengrok/src -d /opengrok/data -H -P -S -G \
    -W /opengrok/etc/configuration.xml -U http://localhost:8080/source

what are those -H -P -S -G for?

**Comment by vladak** [comment]
If you run the indexer with `--help`, it will tell you. Also, see the [wikis](https://github.com/oracle/opengrok/wiki).
